### PR TITLE
[ci] Add more tests to attrib controller tests

### PR DESCRIPTION
**Pull request by eduardoj**
'update' and 'delete' actions are now tested.

### CODE REVIEW

File: src/api/spec/controllers/webui/attributes_spec.rb [hunk 1]
```diff
@@ -54,4 +54,49 @@
...
+  describe 'DELETE #destroy' do
+    let!(:attrib) { create(:attrib, project: user.home_project) }
+
+    before do
+      login user
+    end
+
+    it 'deletes the attrib' do
+      expect {
+        delete :destroy, params: { id: attrib.id }
```
Comment (DavidKang) [inline src/api/spec/controllers/webui/attributes_spec.rb]: Why didn't you put the delete request in the before method?

File: src/api/spec/controllers/webui/attributes_spec.rb [hunk 2]
```diff
@@ -54,4 +54,49 @@
...
+      }.to change{Attrib.count}.by(-1)
```
Comment (eduardoj) [inline src/api/spec/controllers/webui/attributes_spec.rb]: @DavidKang I wanted to check that the `delete` decreased the count by 1. And it is not allowed to put an `expect` in a `before` clause.

Comment (DavidKang) [inline src/api/spec/controllers/webui/attributes_spec.rb]: @eduardoj, but if you put a ` let(:attrib_count) { Attrib.count }`  you could expect that the result are the same, right?

Comment (eduardoj) [inline src/api/spec/controllers/webui/attributes_spec.rb]: @DavidKang Sorry, I don't get it. What would be the `expect` clause you propose?

Comment (DavidKang) [inline src/api/spec/controllers/webui/attributes_spec.rb]: Uhmmm, maybe something like
 ` it { expect(Attrib.where(id: attrib.id)).to eq([]) }` or ` it expect(Attrib.count).not_to eq(attrib_count)`

By the way, I'm fine with it 😄

**Pull request merged**